make/golang.mk

```diff
@@ -20,6 +20,7 @@ GO_LDFLAGS += -X $(VERSION_PACKAGE).GitVersion=$(VERSION) \
 	-X $(VERSION_PACKAGE).GitCommit=$(GIT_COMMIT) \
 	-X $(VERSION_PACKAGE).GitTreeState=$(GIT_TREE_STATE) \
 	-X $(VERSION_PACKAGE).BuildDate=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ') \
```

Comment:
Seems that these variables are not used? Can we delete them?
$(VERSION_PACKAGE).GitVersion
$(VERSION_PACKAGE).GitCommit
$(VERSION_PACKAGE).GitTreeState
$(VERSION_PACKAGE).BuildDate

Reply:
Done, I was trying to add more info from build, but it seems like we do not need those additional info.